tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
```

Comment:
@all3fox 
For note, cross-test case is always failing in unit test because some kind of bugs, so I can't add them 😞